esp32/modules/launcher.py

```diff
@@ -100,13 +98,42 @@ def perform_uninstall(ok):
 
 populate_it()
 
-ugfx.input_attach(ugfx.BTN_A, run_it)
-ugfx.input_attach(ugfx.BTN_SELECT, uninstall_it)
-
-ugfx.input_attach(ugfx.JOY_UP, lambda pushed: ugfx.flush() if pushed else 0)
-ugfx.input_attach(ugfx.JOY_DOWN, lambda pushed: ugfx.flush() if pushed else 0)
+def input_a(pressed):
```

Comment:
These generic `input_x` seem like unnecessary/confusing indirection to me, I'd personally prefer a small lambda when attaching the input handler. Might be a matter of taste, so OK.